From: Debian/Kubuntu Qt/KDE Maintainers <debian-qt-kde@lists.debian.org>
Date: Wed, 13 Jul 2016 05:35:22 +0200
Subject: kubuntu_disable-failing-test

Description: ignore failing tests
 ignore tests failing due to switch to plasma 5
Author: Jonathan Riddell
Origin: me
Forwarded: nope
Last-Update: 2015-09-01
---
 lib/konq/tests/CMakeLists.txt | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/lib/konq/tests/CMakeLists.txt b/lib/konq/tests/CMakeLists.txt
index 9affbf8..8adf107 100644
--- a/lib/konq/tests/CMakeLists.txt
+++ b/lib/konq/tests/CMakeLists.txt
@@ -14,13 +14,4 @@ kde4_add_unit_test(konqpopupmenutest konqpopupmenutest.cpp)
 target_link_libraries(konqpopupmenutest konq ${KDE4_KFILE_LIBS}
                      ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTTEST_LIBRARY})
 
-########### favicontest ###############
-
-set(favicontest_SRCS favicontest.cpp)
-qt4_add_dbus_interface(favicontest_SRCS ${CMAKE_CURRENT_SOURCE_DIR}/../favicons/org.kde.FavIcon.xml favicon_interface)
-kde4_add_unit_test(favicontest ${favicontest_SRCS})
-
-target_link_libraries(favicontest konq ${KDE4_KDECORE_LIBRARY} ${KDE4_KIO_LIBRARY} ${QT_QTCORE_LIBRARY}
-                      ${QT_QTGUI_LIBRARY} ${QT_QTDBUS_LIBRARY} ${QT_QTTEST_LIBRARY})
-
 ############################################
